css/style.css

```diff
@@ -186,8 +186,9 @@ header {
   font-weight: bold;
 }
 
-.below-head {
-  margin-left: 3vw;
+.banner-text .below-head {
+  margin: 50px 0 0 3vw;
+  font-size: 3.2vw;
```

Comment:
On keeping this size it looks huge and there is no much differentiation between the "We are programmers" and the subtitle and even the alignment looks a bit off, try to adjust it.

Reply:
Ok Sir